### PR TITLE
hermes-client adds default headers

### PR DESCRIPTION
**Pull request by adamdubiel**

### CODE REVIEW

**Comment by druminski** [comment]
:+1:

File: hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesMessage.java [hunk 1]
```diff
@@ -81,8 +79,19 @@ public static Builder hermesMessage(String topic, String content) {
...
     }
 
-    static HermesMessage appendContentType(HermesMessage message, String contentType) {
-        message.headers.put(CONTENT_TYPE_HEADER, contentType);
+    /**
+     * This method modifies the state of HermesMessage in order to avoid additional
+     * allocation when appending default values. Using same HermesMessage in multiple
+     * HermesClient objects with different defaults is very unlikely, as is sending the same
+     * message in multiple threads (which could cause issues with concurrent modification of
+     * map.
```
Comment (chemicL) [inline hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesMessage.java]: just unclosed bracket here

File: hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesMessage.java [hunk 2]
```diff
@@ -81,8 +79,19 @@ public static Builder hermesMessage(String topic, String content) {
...
+     */
+    static HermesMessage appendDefaults(HermesMessage message, Map<String, String> headers) {
+        for(Map.Entry<String, String> entry : headers.entrySet()) {
```
Comment (chemicL) [inline hermes-client/src/main/java/pl/allegro/tech/hermes/client/HermesMessage.java]: another minor thing: space after java keywords

**Comment by chemicL** [comment]
:+1:

**Pull request merged**